hookdocs/change-supported-post-types.md

```diff
@@ -0,0 +1,72 @@
+By default, the Sophi for WordPress plugin only adds support for the standard `post` and `page` post types. This value is passed through a custom filter ([`sophi_supported_post_types`]{@link sophi_supported_post_types}), which allows you to customize those values. See below for a few examples of how to utilize this filter.
+
+A few things to note:
+
+1. Where you add this code will depend on your individual setup. Often it makes sense to drop these in your main `functions.php` file
```

Comment:
@dkotter if the user is using a theme from the WordPress theme repository, on update the changes will be lost.
Do you think we can suggest to make a plugin instead?

Reply:
I didn't want to put this in at all but it was a question the Sophi team had on where code changes need to be made. My assumption is anyone using this plugin and needing to customize it will be using a custom theme but that may not be the case. I can modify this a bit to mention putting changes in a custom plugin if not using a custom theme